hand-notebook.ipynb

```diff
@@ -62,28 +62,12 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": null,
    "id": "4b3dc5ed-c48d-45ab-a9ad-f56a79dc1c07",
    "metadata": {
     "tags": []
    },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "https://glo-30-hand.s3.us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N41_00_W087_00_HAND.tif\n",
-      "https://glo-30-hand.s3.us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N41_00_W088_00_HAND.tif\n",
-      "https://glo-30-hand.s3.us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N41_00_W089_00_HAND.tif\n",
-      "https://glo-30-hand.s3.us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N42_00_W087_00_HAND.tif\n",
-      "https://glo-30-hand.s3.us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N42_00_W088_00_HAND.tif\n",
-      "https://glo-30-hand.s3.us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N42_00_W089_00_HAND.tif\n",
-      "https://glo-30-hand.s3.us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N43_00_W087_00_HAND.tif\n",
-      "https://glo-30-hand.s3.us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N43_00_W088_00_HAND.tif\n",
-      "https://glo-30-hand.s3.us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N43_00_W089_00_HAND.tif\n"
-     ]
-    }
-   ],
+   "outputs": [],
```

Comment:
was dropping the output urls intentional? I'd left them in so you get to see example output when browsing the notebook at https://github.com/asjohnston-asf/agu-2022-notebooks/blob/main/hand-notebook.ipynb

Reply:
Nah, I accidentally blew them away. I'll put them back; just need to edit it by hand (or get the STAC catalog up and running and run it!)